client/src/components/workspace/Workspace.jsx

```diff
@@ -26,14 +26,13 @@ export default ({ location: { workspaceId, neighborhood }, details = null } ) =>
 
   let { amenities, photo, description, rates } = space;
 
-
   // conditionally render array of amenities data
-  const Amenities = ({data}) => {
-    if (!data) return <></>
-      const rest = data.amenities.length - 5;
+  const Amenities = ({ amenities: { amenities }}) => {
```

Comment:
Does this work? If amenities is an array, you are creating an object with this structure:
```
{
  amenities: {
    amenities: [...]
  }
}
```

Reply:
This was inherited, and I'm guessing based on the structure of the data he was getting from that service. I'm gonna change it to be more clear though